scripts/zones/Grauberg_[S]/mobs/Scitalis.lua

```diff
@@ -1,9 +1,25 @@
 -----------------------------------
 -- Area: Grauberg [S]
 --   NM: Scitalis
+-- https://www.bg-wiki.com/ffxi/Scitalis
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:addImmunity(xi.immunity.GRAVITY)
+    mob:addImmunity(xi.immunity.SILENCE)
+    mob:addImmunity(xi.immunity.SLOW)
+    mob:addImmunity(xi.immunity.DARK_SLEEP)
+    mob:addImmunity(xi.immunity.PETRIFY)
+
+    mob:setMod(xi.mod.DOUBLE_ATTACK, 50)
+    mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
+end
+
+entity.onAdditionalEffect = function(mob, target, damage)
+    return xi.mob.onAddEffect(mob, target, damage, xi.mob.ae.ENAERO, { power = math.random(165, 190) })
```

Comment:
sources vary greatly on the power of its wind effect, I've seen as low as 120 reported. Put a TODO here to verify the real base power later (we want generally to let resistance functions handle the changes, but right now all the mob AEs are using randomness and haven't been reworked to do this accurately)

Reply:
This was verified via capture last week in retail

Reply:
values seen one exactly one cap with one set of gear does not equal knowing its exact correct base power unmodified. if you had it down, there would not be any math.random() happening

I went out there my self on 2 diff chars and one got walloped for 200 and then other for less than 140. you  have a random between 165 and 190. SOMETHING is influencing the hit, we dunno what is involved yet. Add the TODO.

Reply:
fair enough, here's some data that i just gathered from the cap:

![image](https://github.com/LandSandBoat/server/assets/131182600/84ba75cd-0330-4974-97b6-236b8afebdc4)

raw:

![image](https://github.com/LandSandBoat/server/assets/131182600/ba52a4cf-5379-43d7-bcae-f033e66a9e04)

Reply:
usually these things are heavily influenced by elemental resist and I thought I had it all figured out especially on items before item levels came along and dumped gobs of meva and magic damage stats onto everything and started resulting in wildly different numbers than 1-75.  _cries in math_

Reply:
TODO added, btw